vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -270,7 +271,7 @@ void remove_slot_features(example* shared, example* slot)
       continue;
     }
 
-    else if (index == default_namespace)  // slot default namespace has a special namespace in shared
```

Comment:
I guess it doesn't matter either way, but why remove this?

Reply:
The specific clang-tidy check here is to not follow a `continue` statement with an `else if` as it is kind of misleading